### DISCUSSION TITLE
Control 1 fecha

### DISCUSSION THREAD

**Discussion by Krmn390** [IIC2513/Syllabus#18] [General]
Hola! Estoy confundida, ¿ cuándo van a liberar el control 1?

**Comment by aaossa** [comment]
Hola @Krmn390 depende de tu sección. Al menos en la sección 1 el control siempre ha estado programado para liberarse el 31 de marzo. En cuanto a la sección 2, el control 1 se realizó al final del taller del jueves pasado 👌

**Comment by Krmn390** [comment]
Muchas gracias!!
